jupyter_ydoc/ydoc.py

```diff
@@ -189,6 +201,24 @@ def set(self, value):
             self._ymeta.set(t, "nbformat", nb["nbformat"])
             self._ymeta.set(t, "nbformat_minor", nb["nbformat_minor"])
 
+    def on_cells_changed(self, event):
+        if len(event.target) == 0:
+            # We can not update a YItem from the callback
+            # create a task to do it later
+            asyncio.create_task(self.append_new_cell())
+
+    async def append_new_cell(self):
+        self.append_cell(
+            {
+                "cell_type": "code",
+                "execution_count": None,
+                "metadata": {},
+                "outputs": [],
+                "source": "",
+                "id": str(uuid4()),
+            }
```

Comment:
Worth not repeating with:
https://github.com/jupyter-server/jupyter_ydoc/blob/97fe54bc3a4bebb8134d11c325abe4e1d39bc3ce/jupyter_ydoc/ydoc.py#L178-L185?

Reply:
Why? It creates a new uuid. It will be a different cell.